.flake8

```diff
@@ -32,5 +31,5 @@ exclude=
 
 per-file-ignores =
     .github/workflows/pr_title_tester.py:E501,W605
-    sbin/run_lichen_plagiarism.py:E501
+    sbin/submitty_daemon_jobs/submitty_jobs/jobs.py:E501,E222
```

Comment:
Can we disable `E222` only in the specific places it's needed?

Reply:
The only issue is lines 294-294.  I tried to add an inline ignore but it's several different lines so I'm not really sure how to resolve the warning.

Reply:
https://stackoverflow.com/questions/46795404/how-to-flake8-ignore-in-multiline-code

Reply:
I tried what the comment in the SO post suggested and it didn't work, whoever, it's a little cursed but if I put the inline ignore after each line it resolved it:
```python
self.add_permissions_recursive(split_path,
                                           stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR |   stat.S_IRGRP | stat.S_IWGRP | stat.S_IXGRP |   stat.S_ISGID,  # noqa: E222
                                           stat.S_IRUSR | stat.S_IWUSR | stat.S_IXUSR |   stat.S_IRGRP | stat.S_IWGRP | stat.S_IXGRP |   stat.S_ISGID,  # noqa: E222
                                           stat.S_IRUSR | stat.S_IWUSR |                  stat.S_IRGRP | stat.S_IWGRP)  # noqa: E222
```